.github/workflows/run_system_tests.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         uses: actions/checkout@v3
       - name: Install dependencies
         run: poetry install
+      - name: check installdriver subcommand can be invoked
+        run: poetry run nidaqmx installdriver --help
```

Comment:
The goal of `run_system_tests.yml` is to run Tox, which automatically iterates over multiple Python versions. Move this to `run_unit_tests.yml`, so it is run with each Python version.